spyderlib/plugins/shortcuts.py

```diff
@@ -7,179 +7,505 @@
 """Shortcut management"""
 
 from __future__ import print_function
+import os
+import re
+import sys
 
-from spyderlib.qt.QtGui import (QVBoxLayout, QComboBox, QItemDelegate,
-                                QTableView, QMessageBox, QPushButton)
-from spyderlib.qt.QtCore import Qt, QSize, QAbstractTableModel, QModelIndex
+from spyderlib.qt.QtGui import (QVBoxLayout, QTableView, QMessageBox,
+                                QPushButton, QKeySequence, QDialog,
+                                QDialogButtonBox, QLabel, QGridLayout,
+                                QLineEdit, QAbstractItemView,
+                                QSortFilterProxyModel, QStyledItemDelegate,
+                                QStyleOptionViewItemV4, QApplication,

```

Comment:
It seems there's nothing named `QStyleOptionViewItemV4` in Qt5. There is only `QStyleOptionViewItem`, so this PR is failing on Qt5.